static/index.html.template

```diff
@@ -12,6 +12,9 @@
   </head>
   <body>
     <div id="content"></div>
-    <script src="/static/build/main.bundle.js"></script>
+    <script
+      src="/static/build/main.bundle.js?{{'static/build/main.bundle.js' | md5sum}}"
```

Comment:
@stefanv is the vertical bar jinja-specific? I'm not sure what it achieves here

Reply:
Yes, that is the extension I wrote into our template parser.  It takes the filename, calculates its md5sum, and replaces the `{{ ... }}` with that value.